app/controllers/ticket_purchases_controller.rb

```diff
@@ -4,8 +4,15 @@ class TicketPurchasesController < ApplicationController
   authorize_resource :conference_registrations, class: Registration
 
   def create
+    if(params[:tickets].nil?)
```

Comment:
but in this action you want to create a ticket. What could you pay if there is nothing? :thinking: